_rules/document-has-headings-for-non-repeated-content-047fe0.md

```diff
@@ -0,0 +1,448 @@
+---
+id: 047fe0
+name: Document has heading for non-repeated content
+rule_type: atomic
+description: |
+  This rule checks that the non-repeated content contains a heading
+accessibility_requirements:
+  wcag-technique:H69: # Providing heading elements at the beginning of each section of content
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility Tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+  assets:
+    - _The Three Kingdoms_ by Luo Guanzhong, translation by Yu Sumei (Tuttle publishing, 2014, ISBN 9780804843935)
+    - Image from a Ming Dynasty edition of the Romance of the Three Kingdoms, original kept in the library holdings of Peking University.
+---
+
+## Applicability
+
+This rule applies to any [HTML web page][].
+
+## Expectations
```

Comment:
It seems rather counter-intuitive that a page with no repeating content fails this rule, even if it has headings, just because there is no  content "after repeated content". That seems wrong.